packages/cli/src/oclif/commands/register.js

```diff
@@ -63,37 +98,77 @@ class RegisterCommand extends ZapierBaseCommand {
     }
   }
 
+  /**
+   * Prompts user for values that have not been provided
+   * Flags can heavily impact the behavior of this function
+   * @returns {object}
+   */
   async _promptForAppMeta() {
     const appMeta = {};
 
+    const actionChoices = [
+      { name: 'Update current integration', value: 'update' },
+      { name: 'Register a new integration', value: 'register' },
+    ];
+    appMeta.action = actionChoices[1].value; // Default action is register
+    if (this._hasAppFile()) {
+      console.info(colors.yellow('.zapierapprc file detected.'));
+      if (this.flags.yes) {
+        console.info(
+          colors.yellow('-y/--yes flag passed, updating current integration.')
+        );
+        appMeta.action = actionChoices[0].value;
+      } else {
+        appMeta.action = await this.promptWithList(
+          'Would you like to update your current integration or register a new one?',
```

Comment:
The `--yes` help text says "assume yes for all yes/no prompts", but this isn't a clear yes-or-no question. Maybe we could rephrase the question and add an app ID to make it more clear?

```
          'Would you like to update your current integration (ID: {app_id})?',
```

And in the choices, we can say:

```
      { name: 'Yes, update current integration', value: 'update' },
      { name: 'No, register a new integration', value: 'register' },
```

Reply:
@eliangcs Good call! Updated in 4594d1c5bd2ba95426e92e212a056ca95f0f3969